src/lib.rs

```diff
@@ -380,4 +380,10 @@ mod tests {
         assert_eq!(Tz::UTC.name(), "UTC");
         assert_eq!(Tz::Zulu.name(), "Zulu");
     }
+
+    #[test]
+    fn test_impl_hash() {
+        #[derive(Hash)]
```

Comment:
Could you throw an `#[allow(dead_code)]` in here so that we don't get compiler warnings every time we test?